uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-client/src/main/java/org/uberfire/ext/widgets/core/client/workbench/widgets/popups/activities/notfound/ActivityNotFoundPresenter.java

```diff
@@ -36,41 +35,29 @@
 public class ActivityNotFoundPresenter {
 
     @Inject
-    private View view;
+    private ActivityNotFoundView view;
+
     @Inject
     private PlaceManager placeManager;
+
     private PlaceRequest place;
 
     @OnStartup
     public void onStartup(final PlaceRequest place) {
         this.place = place;
     }
 
-    @OnOpen
-    public void onOpen() {
-        final String identifier = place.getParameter("requestedPlaceIdentifier",
-                                                     null);
-        view.setRequestedPlaceIdentifier(identifier);
-    }
-
     @WorkbenchPartTitle
     public String getTitle() {
         return CoreConstants.INSTANCE.ActivityNotFound();
     }
 
     @WorkbenchPartView
-    public UberView<ActivityNotFoundPresenter> getView() {
```

Comment:
@jomarko Sorry to come with this late to the party, but what is the reasoning behind this change? Why use pure simple panel ?

Reply:
Because as we decided not to show place identifier in the popup message, we don't need the presenter more to be injected into view.

Reply:
@jomarko I prefer to keep MVP architecture. Could you please update the PR? Thanks.